ShibbolethAuthPlugin.inc.php

```diff
@@ -204,7 +205,19 @@ function getEnabled($contextId = null) {
 		return $this->getSetting($this->_contextId, 'enabled');
 	}
 
-
+        /**
+         * @copydoc Plugin::isShibbolethConfigured()
+         * Determine whether or not this plugin is currently configured.
+         * @return boolean
+         */
+        function isShibbolethConfigured() {
+		foreach($this->settingsRequired as $setting){
```

Comment:
Spacing here should match the surrounding style.  That is, spaces between the keyword and open parenthesis, and between the close parenthesis and open brace.